chapters/ch06/3.spell-correction.ipynb

```diff
@@ -746,23 +723,44 @@
        "      <td>2547</td>\n",
        "      <td>1</td>\n",
        "    </tr>\n",
+       "    <tr>\n",
+       "      <th>226</th>\n",
+       "      <td>mcbook</td>\n",
+       "      <td>macbook</td>\n",
+       "      <td>6</td>\n",
+       "      <td>2534</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>132</th>\n",
+       "      <td>ho tablet</td>\n",
+       "      <td>hp tablet</td>\n",
+       "      <td>6</td>\n",
+       "      <td>1996</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>145</th>\n",
+       "      <td>hrc flyer</td>\n",
+       "      <td>htc flyer</td>\n",
+       "      <td>5</td>\n",
+       "      <td>1934</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "        misspell   correction  misspell_counts  correction_counts  edit_dist\n",
+       "     misspelling   correction  misspell_counts  correction_counts  edit_dist\n",
        "168        ipad.         ipad                6               7749          1\n",
        "142   hp tochpad  hp touchpad                6               7144          1\n",
        "143   hp toucpad  hp touchpad                6               7144          1\n",
        "176    iphone s4    iphone 4s                5               4642          2\n",
        "179    iphone4 s    iphone 4s                5               4642          2\n",
-       "180   iphones 4s    iphone 4s                5               4642          1\n",
```

Comment:
Do you know why the output is so different vs. before? Was it a library upgrade or something?